service-commands/src/commands/service-commands.json

```diff
@@ -41,11 +41,26 @@
       "cd contracts/; npm run ganache-q"
     ]
   },
+  "contracts-predeployed": {
+    "up": [
+      "docker run --name audius_ganache_cli -d -p 8545:8545 --network=audius_dev audius/ganache:data-contracts-predeployed-4d4a2073fb18b071f7dcf01fdcc4b26001162f48",
+      "docker cp audius_ganache_cli:/app/contract-config.json $PROTOCOL_DIR/identity-service/contract-config.json",
+      "docker cp audius_ganache_cli:/app/contract-config.json $PROTOCOL_DIR/creator-node/contract-config.json",
+      "docker cp audius_ganache_cli:/app/contract-config.json ~/.audius/contract-config.json",
+      "docker cp audius_ganache_cli:/app/contract_config.ini $PROTOCOL_DIR/discovery-provider/contract_config.ini",
+      "docker cp audius_ganache_cli:/app/contracts/. $PROTOCOL_DIR/discovery-provider/build/contracts",
+      "docker cp audius_ganache_cli:/app/contracts/. $PROTOCOL_DIR/libs/data-contracts/ABIs",
+      "docker cp audius_ganache_cli:/app/build/contracts/. $PROTOCOL_DIR/contracts/build"
+    ],
+    "down": [
+      "cd contracts/; npm run ganache-q"
+    ]
+  },
   "eth-contracts": {
     "path": "eth-contracts",
     "up": [
       "cd eth-contracts/; sudo rm -rf build",
-      "docker run --name audius_ganache_cli_eth_contracts -d -p 8546:8545 --network=audius_dev trufflesuite/ganache-cli:v6.9.1 -h 0.0.0.0 --acctKeys eth-contracts-ganache-accounts.json -a 100 -l 8000000 --deterministic true",
+      "docker run --name audius_ganache_cli_eth_contracts -d -p 8546:8545 --network=audius_dev trufflesuite/ganache-cli:v6.9.1 -h 0.0.0.0 --acctKeys eth-contracts-ganache-accounts.json -a 100 -l 8000000 --networkId 1000000000000 --deterministic true --db ./db",
```

Comment:
Docs on network id: `Specify the network id ganache-cli will use to identify itself (defaults to the current time or the network id of the forked blockchain if configured)`

If the network id were to change every time, the previously migrated contracts would not work because the deployed contracts take in network id as a param of the constructor and is used to verify signed data - so if we restart ganache, it gets a new network id and signing data fails becuase it has a new network id
(Took me a while to figure this out)